### PR TITLE
ThrownItem to ThrownProjectile

### PR DESCRIPTION
**Pull request by xNexus-ACS**

### CODE REVIEW

**Review by iopietro — APPROVED** [review]

**Review by louis1706 — CHANGES_REQUESTED** [review]
ThrownProjectile

**Comment by iopietro** [comment]
Yeah, we could call it like that then.

**Review by iopietro — CHANGES_REQUESTED** [review]
Call it ThrownProjectile

**Comment by xNexus-ACS** [comment]
Sure

**Comment by xNexus-ACS** [comment]
Done guys

**Review by iopietro — APPROVED** [review]

**Comment by joker-119** [comment]
Prs should always target dev not master

File: .github/workflows/programs.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        proj_name: [Exiled.Installer, Exiled.Patcher]
+        proj_name: [Exiled.Installer]
```
Comment (Thundermaker300) [inline .github/workflows/programs.yml:15]: ?

Comment (iopietro) [reply .github/workflows/programs.yml:15]: It's from Joker's commit.

Comment (xNexus-ACS) [reply .github/workflows/programs.yml:15]: yeah

**Review by iopietro — APPROVED** [review]

File: .github/workflows/programs.yml (hunk 1 above)
Comment (Thundermaker300) [reply .github/workflows/programs.yml:15]: got it

**Review by iopietro — APPROVED** [review]

**Review by Thundermaker300 — APPROVED** [review]

**Pull request merged**